scene/main/viewport.cpp

```diff
@@ -655,9 +655,7 @@ void Viewport::_notification(int p_what) {
 		case NOTIFICATION_WM_WINDOW_FOCUS_OUT: {
 			_gui_cancel_tooltip();
 			_drop_physics_mouseover();
-			if (gui.mouse_focus && !gui.forced_mouse_focus) {
```

Comment:
Shouldn't the `gui.mouse_focus` check stay?

Reply:
I put the check inside `_drop_mouse_focus`, since it is used without a check in `set_disable_input`. 
This also helps clean up `mouse_focus_mask` if they get out of sync somehow.